include/mantella_bits/orbitalMechanics.hpp

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+// C++ standard library
+#include <utility>
+#include <vector>
+
+// Armadillo
+#include <armadillo>
+
+namespace mant {
+  namespace itd {
+
+    // from pykep:
+    // ref_mjd2000(epoch(2451545.0,epoch::JD).mjd2000()
+    //inline double jd2mjd2000(const double & in){
+    //return ( in - 2451544.5 );
+    //}
+    // => 2451545.0 - 2451544.5
+    const double refModifiedJulianDay2000 = 0.5;
+
+    double lambda;
+
+    const double au = 149597870691.0;
+    const double muSun = 1.32712440018e20;
+
+    const int lambertMaxIterations = 13;
+    const double lambertTolerance = 1e-13;
+
+    double degreeToRadian(
+        const double degree);
+
+    std::vector<std::pair<arma::Col<double>::fixed<3>, arma::Col<double>::fixed<3>>> lambert(
+        const arma::Col<double>::fixed<3>& departurePosition,
+        const arma::Col<double>::fixed<3>& arrivalPosition,
+        const double transferTime,
+        const bool clockwiseDirection);
```

Comment:
clockwise is already a direction, naming it `clockwise` would also be fine. However, adding an `is` prefix to booleans is even better. (`isClockwise`).

But if I remember it correctly, this parameter will vanish, won't it?